dev/com.ibm.ws.jaxws.2.3.common/bnd.bnd

```diff
@@ -53,7 +53,6 @@ Import-Package: \
 
 Private-Package:\
    com.ibm.ws.jaxws.*, \
-   com.ibm.ws.jaxws23.component, \
```

Comment:
This package doesn't exist, I added it to test a fix but removed it when it had no effect